### PR TITLE
Use informers for CVO

### PR DESCRIPTION
**Pull request by abhinavdahiya**
Uses informer pattern for CVO
/assign @crawford

### CODE REVIEW

File: cmd/start.go [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+	rootCmd.AddCommand(startCmd)
+	startCmd.PersistentFlags().StringVar(&startOpts.kubeconfig, "kubeconfig", "", "Kubeconfig file to access a remote cluster (testing only)")
+}
+
+func runStartCmd(cmd *cobra.Command, args []string) {
+	flag.Set("logtostderr", "true")
+	flag.Parse()
+
+	// To help debugging, immediately log version
+	glog.Infof("Version: %+v", version.Version)
```
Comment (crawford) [inline cmd/start.go]: Use `version.String` instead.

Comment (abhinavdahiya) [reply cmd/start.go]: :+1: done

**Pull request merged**